### ISSUE TITLE
'timestamp with time zone' for sql definition

### ISSUE THREAD

**Issue by stanislavt** [db-migrate/shared#1]
Hi!
Will it possible to add one more datatype? I want to use 'timestamp with time zone' datatype for `created` and `lastupdated` columns. For my project I have used it like: 
`
    created: 'timestamp with time zone',
    lastupdated: 'timestamp with time zone'
`
but it calls a warning in console: `[WARN] Using unknown data type TIMESTAMP WITH TIME ZONE`. 
I think the new datatype in the datatypes would be perfect! 
##

**Comment by wzrdtales** [comment]
No. This is driver dependend, not shared depended. It would be shared if this would apply to **all** databases though.

Please refer to the driver this belongs to, also are you sure that this works like you described it? And you should also not that this is not a `dataType` anymore. The `dataType` actually would be `timestamp` and `with time zone` is an option, but not a `dataType`.

**Comment by stanislavt** [comment]
Ok, thank you! 

Could you tell me please, how can I Add this option? 
`db.createTable('airline', {
    id: {type: 'int', primaryKey: true, autoIncrement: true},
    name: 'string',
    created: 'timestamp'
  }, cb);`

**Comment by stanislavt** [comment]
Can I add index for column while creating the new table?

**Comment by wzrdtales** [comment]
Only primary keys currently, so no not as of now.

To your other question: Not  without having a new constraint added to the pg driver.
https://github.com/db-migrate/pg/blob/master/index.js#L268

**Comment by stanislavt** [comment]
Thanks a lot!

**Comment by wzrdtales** [comment]
@stanislavT No problem, if you plan to go for a PR on the postgres driver, please feel welcome to do so. I'm always here and open for opinions and questions. If you need or want to know anything, just let me know and I will see what I can do.
